CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 * Remove deprecated `Kafka.spec.topicOperator` classes and deployment logic
 * Use Java 11 as the Java runtime
+* Upgrade to Cruise Control version 2.0.108
```

Comment:
I'm not sure this is worth mentioning. If something, I would probably rather mention what makes this upgrade interesting for users.

Reply:
Worth mentioning that they no longer need to create the `strimzi.cruisecontrol.metrics` topic before deployment if `auto.create.topics.enable: false` is configured.